### PR TITLE
rails: add page

### PR DESCRIPTION
**Pull request by moalamin**
----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/rails.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# rails
+
+> Ruby on Rails, or Rails, is a server-side MVC framework written in Ruby under the MIT License.
```
Comment (agnivade) [inline pages/common/rails.md]: Change this to `A server-side MVC framework written in Ruby`

File: pages/common/rails.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Check current version of rails:
```
Comment (agnivade) [inline pages/common/rails.md]: Make this as the last example.

Tip - Think of the natural order of usage that a user is going to do. First thing would be to create the project, start server and do other stuff. And then at last, there might be a need to check the version.

File: pages/common/rails.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`rails --version`
+
+- Create a new rails project:
+
+`rails new "{{name_of_project}}"`
```
Comment (agnivade) [inline pages/common/rails.md]: name_of_project => project_name

File: pages/common/rails.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Start local server for current project on port 3000 (while in projects root directory):
```
Comment (agnivade) [inline pages/common/rails.md]: It is well understood that you have to be in project's root directory to execute these commands. We have lot of pages like this and it's not explicitly mentioned anywhere. Unless a tool uses some other convention, we omit it for brevity. 

Let's remove it from all the descriptions.

File: pages/common/rails.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+`rails server`
+
+- Start local server for current project on a specified port (while in projects root directory):
+
+`rails server -p "{{port_number}}"`
```
Comment (agnivade) [inline pages/common/rails.md]: port_number => port

File: pages/common/rails.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+- Open console to interact with applicaiton from command line (while in projects root directory):
```
Comment (agnivade) [inline pages/common/rails.md]: typo in application

**Comment by moalamin** [comment]
Thanks a lot.

**Comment by sbrl** [comment]
Thanks, @moalamin!

**Pull request merged**